### PR TITLE
Various UI fixes and updates

### PR DESCRIPTION
**Pull request by shanejann**
Updated:
- Fonts to "Anonymous Pro", a monospace font that should keep our hashes the same total width https://github.com/HandshakeAlliance/HNScan/issues/105
- Names on the Names List now truncate at mobile instead of wrapping https://github.com/HandshakeAlliance/HNScan/issues/104

Fixed:
- Background color is no longer white on dark theme https://github.com/HandshakeAlliance/HNScan/issues/81
- Svg now fills on the 404 page https://github.com/HandshakeAlliance/HNScan/issues/79

Removed:
- Extraneous "Nunito" font

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "autoprefixer": "^9.3.1",
     "axios": "^0.18.0",
     "boom": "^7.2.2",
-    "browser-sync": "^2.26.3",
+    "browser-sync": "^2.26.5",
```
Comment (shanejann) [inline package.json]: I believe this was a vulnerability fix (using npm audit)

File: src/public/css/base.css [hunk 2]
```diff
@@ -1,8 +1,12 @@
+html {
+  background-color: var(--background);
```
Comment (shanejann) [inline src/public/css/base.css]: so background was already on body and wasn't doing the trick. This one seems to correctly change the background to light-black when in dark theme.

File: src/public/img/3d.svg [hunk 3]
```diff
@@ -1,3 +1,3 @@
-<svg height="20px" width="20px" viewBox="-34 0 512 512" xmlns="http://www.w3.org/2000/svg">
+<svg height="100%" width="100%" viewBox="-34 0 512 512" xmlns="http://www.w3.org/2000/svg">
```
Comment (shanejann) [inline src/public/img/3d.svg]: letting the containers take care of 3d block instead of the svg

File: src/templates/layouts/layout.pug [hunk 4]
```diff
@@ -10,6 +10,7 @@ html(lang="en")
     link(rel='stylesheet', href='https://use.fontawesome.com/releases/v5.4.2/css/all.css', integrity='sha384-/rXc/GQVaYpyDdyxK+ecHPVYJSN9bmVFBvjA/9eOB+pb3F2w2N6fc5qB9Ew5yIns', crossorigin='anonymous')
     link(href="https://fonts.googleapis.com/css?family=Nunito:400,600" rel="stylesheet")
```
Comment (shanejann) [inline src/templates/layouts/layout.pug]: should I kill this? Not sure where it's used (I don't think it is used).

Comment (kilpatty) [reply src/templates/layouts/layout.pug]: Yeah kill this

**Comment by kilpatty** [comment]
LGTM if you want to kill that extra font and then merge 👍

**Pull request merged**